src/Microsoft.AspNet.Http/DefaultHttpRequestIdentifierFeature.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+
+namespace Microsoft.AspNet.Http.Features.Internal
+{
+    public class DefaultHttpRequestIdentifierFeature : HttpRequestIdentifierFeature, IHttpRequestIdentifierFeature
```

Comment:
There's no reason for DefaultHttpRequestIdentifierFeature to exist independent of HttpRequestIdentifierFeature, just merge them.